admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -220,25 +225,43 @@ Upgrade on CentOS/Red Hat
 
 .. _upgrade-search-indices:
 
-Upgrade search indices
-----------------------
+Upgrade Elasticsearch and search indexes
+----------------------------------------
 
 .. note::
 
-   Ignore this section if you are planning to run Archivematica without a
-   search index. Instead, follow the instructions on :ref:`how
-   to upgrade Archivematica in indexless mode <upgrade-indexless>`.
+   Ignore this section if you are planning to run Archivematica without search
+   indexes. Instead, follow the instructions on :ref:`how to upgrade
+   Archivematica in indexless mode <upgrade-indexless>`.
+
+Archivematica |release| uses Elasticsearch 6.x as its search engine. If you're
+upgrading from Archivematica 1.8.x or lower, where Elasticsearch 1.x was the
+supported version, it's required to upgrade your Elasticsearch cluster and
+indexes to the new version.
 
-To complete these instructions you need to know the paths of your transfer
-backlog and AIP storage locations. Respectively, these usually are:
+To complete this upgrade is important to know if you have access to your
+transfer backlog and AIP storage locations in the local filesystem.
+Respectively, these usually are located in the following paths:
 
 * :file:`/var/archivematica/sharedDirectory/www/AIPsStore/transferBacklog`
 * :file:`/var/archivematica/sharedDirectory/www/AIPsStore`
 
-You can confirm the paths of your installation in the Location tab of the
+You should confirm the paths of your installation in the Location tab of the
 Storage Service.
 
-This process has two steps:
+If you have access to those locations, the recommended method for the upgrade is
```

Comment:
We should include here something like "and the space used is less than 1 terabyte" or something like that, so the reindex method is not used when big aipstores are in place.

Reply:
Good point @scollazo!

After the discussion about changing the maintenance section I have moved the command details to that page and linked them from here. I have added notices in both places about the duration of those commands for big storage locations, but I think we should avoid adding an specific storage size.